jenkins/release-tag/release-tag.jenkinsfile

```diff
@@ -0,0 +1,57 @@
+lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+
+pipeline {
+    options {
+            timeout(time: 2, unit: 'HOURS')
+    }
+    agent {
+        docker {
+            label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:centos7-x64-arm64-jdkmulti-node10.24.1-cypress6.9.1-20211130'
```

Comment:
Try to see if you can use `alpine:latest` instead of this full ledge image for testing.
Since this is literally a git command pipeline.

Reply:
Eh, seems like you need aws cli as well.